### DISCUSSION TITLE
Uncovering the Action Colors used in Wasm

### DISCUSSION THREAD

**Discussion by johanvdmeer** [b-inary/wasm-postflop#6] [General]
I would like to know which colors are used for all actions from preflop to the river in the hand-matrix? 

If I try to find this out by using page source to display or inspect it will be quite a lot of work to figure out. 

Do you have any advice for me, thank you in advance?

Johannes

**Comment by b-inary** [comment]
Currently, color definitions are scattered throughout the source code. Color names are based on the [Tailwind CSS convention](https://tailwindcss.com/docs/customizing-colors).
- `yellow500` is used for preflop ranges. The variable `yellow500` can be found in `RangeEditor.vue` or `RangeMiniViewer.vue`. The background color is `neutral800`, except for the diagonals, which have `neutral700` background color.
- Colors for postflop actions are defined in `ResultNav.vue`. The variables `foldColor` (`blue500`), `checkColor` (`green500`), `callColor` (`green500`), and `betColorGradient` (from `amber500` to `violet500`) are defined here.
